_episodes/04-use-license.md

```diff
@@ -5,7 +5,185 @@ exercises: 0
+
+If you are the creator of an original work, you can decide the licence for it, but you should be careful of some common pitfalls:
+
+* you are contributing to an existent project, in this case your contribution should be licence compatible with the project that you are contributing to;
+* Giving away your copyright as part of your employment contract. This is very common and if this is the case you must seek permission of your employer to use a open source licence;
+* the terms and conditions of the grant funding the project, this is getting more common since funding agencies are requiring research outputs to be open access in which case you must choose a compatible licence.
+
+Since these are only three examples of issues that can arise, we recommend you to talk with the Intellectual Property (IP) office at your institution.
+
+Once added, a licence and all future versions of the software, will be under that licence. Most projects never need to change licenses, but occasionally circumstances change. When changing a project’s license, there is one fundamental thing to consider: **all** project's existing copyright holders should give their agreement in writing and signed. This is complicated because, without a previous copyright transfer agreement, the list of copyright holders can have thousands of names. We recommand to have a look at the [guide from GitHub on this question](https://opensource.guide/legal/#what-if-i-want-to-change-the-license-of-my-project) for more details.
+
+> ### Note: Dual License
+>
+> Some software that have comercial potential adopt a dual license model. They have a version of the code under the GNU General Public License that is freely available to everyone, but they also offer a proprietary version of the code in exchange of some cash. If you decide to adopt this business model, you must be very careful to correctly collect copyright transfer from all contributors to you.
+{: .callout}
+
+In principle, you can create your own bespoke licence, it's your right. However, we strongly discourage it, unless you have appropriate legal background. You are helping the users to understand the licence over your code or data by adopting an existing, widely described licence which better suits with your creation. So, users do not have to analyze the legal details of the licence to understand it.
+
+* Example licences explained in terms of the concepts discussed previously
+https://choosealicense.com/
+
+## Add a LICENCE file to a repository
+
+By including an open source license in your repository, it makes easier for other people to contribute. Most of the projects places their license information in a file named `LICENSE` (`LICENSE.txt` or `LICENSE.md` or `LICENSE.rst` or `COPYING`, `COPYING.txt`, `COPYING.md`, `COPYING.rst` are also used) in the root of the repository.
+
+> ## Challenge 
+> 
+> Add the previously selected license to your GitHub repository
```

Comment:
"the previously selected", selected where? Cannot they change to select a license they like better than the prevoiously selected one?